pkg/controller/instance/instance_controller.go

```diff
@@ -258,6 +276,33 @@ func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	return computeTheReconcileResult(instance, time.Now), nil
 }
 
+func ensureReadinessInitialized(i *kudoapi.Instance) {
+	if i.Spec.PlanExecution.PlanName == kudoapi.DeployPlanName || i.Spec.PlanExecution.PlanName == kudoapi.UpgradePlanName || i.Spec.PlanExecution.PlanName == kudoapi.UpdatePlanName {
+		i.SetReadinessUnknown()
+	}
```

Comment:
```suggestion
	}
	// For any other plan we keep the existing Readiness. As the deploy plan is always the first plan to run, the Readiness is always initialized.
```

It took me a second to understand why a func that "ensures that Readiness is initialized" doesn't set a value in all cases, so, a comment might help.